codecparsers/bitReader_unittest.cpp

```diff
@@ -48,12 +48,12 @@ BITREADER_TEST(NoOverflow)
     std::vector<uint8_t>& data = largeData();
     const uint32_t nBytes = data.size();
 
-    ASSERT_EQ(0, nBytes % (1024 * 1024 * 512));
+    ASSERT_EQ(0u, nBytes % (1024 * 1024 * 512));
```

Comment:
I don't think integer-suffixes are inconvenient.  Plus, they advertise the intentions of the test better.  FYI, these warnings are happening on older GCC versions too.  IMO we shouldn't be ignoring any warnings.